### PR TITLE
Train 120

### PR DESCRIPTION
**Pull request by mellenburg**
Included PRs:
https://github.com/dcos/dcos/pull/1016
https://github.com/dcos/dcos/pull/1096
https://github.com/dcos/dcos/pull/1129
https://github.com/dcos/dcos/pull/1144
https://github.com/dcos/dcos/pull/1115

Downstream Bump:
https://github.com/mesosphere/dcos-enterprise/pull/483

### CODE REVIEW

File: packages/dcos-integration-test/extra/api_session_fixture.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""This file has only one function: to provide a correctly configured
+<<<<<<< HEAD:packages/dcos-integration-test/extra/cluster_fixture.py
```
Comment (mellenburg) [inline packages/dcos-integration-test/extra/api_session_fixture.py]: Missed this merge conflict hidden in comments. Will amend

**Comment by lingmann** [comment]
@amitaekbote there is still a lot of red on that downstream bump.

**Comment by amitaekbote** [comment]
I just noticed, my bad :(  Approved once the enterprise goes green

**Comment by spahl** [comment]
@amitaekbote I think github allows you to withdraw your approval.

**Comment by lingmann** [comment]
@mellenburg if you are going to re-create this train, please include #1115. It is a documentation update and has two ship-its.

**Comment by mellenburg** [comment]
Kicking https://github.com/dcos/dcos/pull/1082 from this train as it needs some EE nuance to handle strict

**Comment by mellenburg** [comment]
@lingmann @spahl This is ready for review. The issue in https://github.com/mesosphere/dcos-enterprise/pull/483 is a strict mode failure that is expected (and should be re-muted until this is merged: https://github.com/mesosphere/dcos-enterprise/pull/409 )

**Pull request merged**